archaius2-commons-configuration/src/main/java/com/netflix/archaius/commons/CommonsToConfig.java

```diff
@@ -38,6 +39,12 @@
     
     public CommonsToConfig(AbstractConfiguration config) {
         this.config = config;
+        config.addConfigurationListener(new ConfigurationListener() {
```

Comment:
I'm really having hard time wrapping my head around all these re-directions. So we're adding a listener here to make sure whoever is depending on CommonsToConfig will have a chance to get notified when underneath config (Archaius1 AbstractConfiguration) changes? Only use case I can think of is one Archaius2 based library being based on another Archaius1 bridged Archaius2 library instead of just directly on the base Archaius1 config? Let me know if I'm wrong, but otherwise, I think we're overloading this bridge. The bridge should just bridge one Archaius1 AbstractConfiguration to another Archaius2 AbstractConfig.